server/authz/policy.rego

```diff
@@ -705,6 +705,13 @@ allow {
   action == [read, write][_]
 }
 
+# Global gitops can write bootstrap packages.
+allow {
+  object.type == "mdm_apple_bootstrap_package"
+  subject.global_role == gitops
+  action == write
+}
```

Comment:
I'll merge so it can make it to the release, and I'm 99.99% sure that's what we want, but @lucasmrod let me know in the off chance that's not the case and I'll push a PR to remove that gitops permission.